src/extension.ts

```diff
@@ -129,6 +130,21 @@ function registerCommands(
          return runDeploy(context, completedSteps, outputChannel);
       }
    );
+
+   registerCommand(
+      'aks-draft-extension.runDraftWorkflow',
+      (
+         actionContext: IActionContext,
+         completedSteps: CompletedSteps | undefined
+      ) => {
+         const context: Context = {actionContext, extensionContext};
+
+         if (completedSteps === undefined) {
```

Comment:
Should just remove the completedSteps here since it's not used at all.

They are used to skip steps if a user directly goes from one step to another. For example, if a user drafts a dockerfile and enters the port then we won't prompt for port again if they directly go to draft a deployment. This functionality isn't used here so it can just be removed.

Reply:
removed